src/common/comp/knowhere_config.cc

```diff
@@ -186,7 +186,7 @@ KnowhereConfig::FreeGPUResource() {
 }
 
 void
-KnowhereConfig::SetRaftMemPool(size_t init_size, size_t max_size) {
+SetRaftMemPoolImpl(std::optional<size_t> init_size, std::optional<size_t> max_size) {
```

Comment:
Do not modify the function definition here, as there is code in Milvus that calls this interface.